applications/nrf5340_audio/src/bluetooth/le_audio_bis_headset.c

```diff
@@ -407,9 +351,7 @@ static void syncable_cb(struct bt_bap_broadcast_sink *sink, bool encrypted)
 	init_routine_completed = true;
 }
 
-static struct bt_bap_broadcast_sink_cb broadcast_sink_cbs = {.scan_recv = scan_recv_cb,
-							     .scan_term = scan_term_cb,
-							     .pa_synced = pa_synced_cb,
+static struct bt_bap_broadcast_sink_cb broadcast_sink_cbs = {.pa_synced = pa_synced_cb,
 							     .pa_sync_lost = pa_sync_lost_cb,
 							     .base_recv = base_recv_cb,
 							     .syncable = syncable_cb};
```

Comment:
Protip when using clang-format: If you add a `,` after the last entry (generally a good idea, as it makes it easier to expand with minimal line changed), then it formats it in (IMO) a more readable way